Mono.Debugging/Mono.Debugging.Client/DebuggerStatistics.cs

```diff
@@ -39,6 +41,11 @@ public class DebuggerStatistics
 		TimeSpan minTime = TimeSpan.MaxValue;
 		TimeSpan maxTime, totalTime;
 
+		public DebuggerStatistics(string name)
+		{
+			Name = name;
+		}
```

Comment:
API break? Should we also add the DebuggerStatistics method which takes no parameter?